gramex/handlers/mlhandler.py

```diff
@@ -348,11 +364,16 @@ def _train(self, data=None):
         data = self._parse_data(False) if data is None else data
         data = self._filtercols(data)
         data = self._filterrows(data)
-        target = data[target_col]
-        train = data[[c for c in data if c != target_col]]
         self.model = self._assemble_pipeline(data, force=True)
-        _fit(self.model, train, target, self.model_path)
-        return {'score': self.model.score(train, target)}
+        if not isinstance(self.model[-1], TransformerMixin):
+            target = data[target_col]
+            train = data[[c for c in data if c != target_col]]
+            _fit(self.model, train, target, self.model_path)
+            result = {'score': self.model.score(train, target)}
+        else:
+            _fit(self.model, data, path=self.model_path)
+            result = {k: getattr(self.model[-1], k) for k in SKLEARN_DECOMPOSE_ATTRS}
```

Comment:
Should we just return whatever attributes `.model` has? This should work with *any* TransformerMixin.